tests/baselines/reference/convertToAsyncFunction/convertToAsyncFunction_CatchAndRejRef.ts

```diff
@@ -1,37 +0,0 @@
-// ==ORIGINAL==
-
-function /*[#|*/f/*|]*/():Promise<void> {
-    return fetch('https://typescriptlang.org').then(res, rej).catch(catch_err)
-}
-function res(result){
-    console.log(result);
-}
-function rej(rejection){
-    return rejection.ok;
-}
-function catch_err(err){
-    console.log(err);
-}
-// ==ASYNC FUNCTION::Convert to async function==
-
-async function f():Promise<void> {
-    try {
-        try {
-            const result = await fetch('https://typescriptlang.org');
-            return res(result);
```

Comment:
What was wrong here?  Does this belong outside the try-catch?

Reply:
In #38152 we removed support for applying the code fix if both an `onfulfilled` and an `onrejected` handler are present for the same call to `.then`, however we never removed some of the code for that refactor and it was inadvertently being called. If you look at both of these tests in https://github.com/microsoft/TypeScript/pull/45536/files/da1d25c4f7460212ba6e40b47ccebb07703f2d63#diff-8eb4fd66983b9bab0f035c7dabf674c0d05e08266a0fd1b0f09dde1ab1a8a840 you'll see that they both have both handlers present.